src/components/clear.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { Colors } from '../styles/colors';
+
+import { StyleSheet, Text, TouchableOpacity, View } from 'react-native';
+
+interface ButProps {
+  text: string;
+  callback: () => void;
+}
+
+export default function Button({ text, callback }: ButProps): React.JSX.Element {
```

Comment:
We prefer to use named export/imports instead of defaults, to avoid importing something wrong or having inconsistent names among the codebase.

```suggestion
export function Button({ text, callback }: ButProps): React.JSX.Element {
```

or

```suggestion
export const Button = ({ text, callback }: ButProps): React.JSX.Element => {
```